specs/eventing/overview.md

```diff
@@ -1,97 +1,133 @@
-
-## Channel
-
-**Channel** provides an event delivery mechanism which can fan out received
-events to multiple destinations via _Subscriptions_. A _Channel_ has a single
-inbound _Addressable_ interface which may accept events delivered directly or
-forwarded from multiple _Subscriptions_. Different _Channels_ may implement
-different degrees of persistence. Event delivery order is dependent on the
-backing implementation of the _Channel_.
-
-Event selection on a _Channel_ is 1:N – a single _Channel_ may fan out to
-multiple _Subscriptions_.
-
-See [Kind: Channel](spec.md#kind-channel).
-
----
-
-_Navigation_:
-
-- [Motivation and goals](motivation.md)
-- **Resource type overview**
-- [Interface contracts](interfaces.md)
-- [Object model specification](spec.md)
+The Knative Eventing API provides primitives for two common event-processing
+patterns:
+
+- Topology-based event routing ([`messaging.knative.dev`](#messaging))
+
+- Content-based event routing ([`eventing.knative.dev`](#eventing))
+
+Knative Eventing does not directly specify mechanisms for other event-processing
+models, including multi-stage workflows, correlated request-reply, and
+sequential (windowed) event processing; these models could be built using the
+primitives provided by Knative, or Knative could deliver events to an external
+system that implements these models.
+
+In addition to the primitives needed to express the above patterns, Knative
+Eventing defines two [_interface contracts_](#interface-contracts) to allow
+connecting multiple types of Kubernetes objects as event senders and recipients
+to the core primitives.
+
+![Overview of objects](images/eventing-overview.svg)
+<!-- Generated from the following UML using PlantUML:
+@startuml
+' Remove shadows
+skinparam shadowing false
+' Set ordering
+left to right direction
+
+usecase Addressable <<Knative Service>>
+usecase Addressable AS A2 <<Kubernetes Service>>
+
+package "eventing.knative.dev" {
+agent Broker
+agent "Trigger" as T1
+agent "Trigger" as T2
+}
+
+Broker <-- T1 : broker
+Broker <-- T2 : broker
+Addressable <-u- T1 : subscriber
+
+package "messaging.knative.dev" {
+agent Channel
+agent "Subscription" as S1
+agent "Subscription" as S2
+}
+
+Channel <-- S1 :channel
+Channel <-- S2 :channel
+Addressable <-u- S2 :subscriber
+A2 <-u- S2 : reply
+@enduml
+-->
+
+## Interface Contracts
+
+In addition to the concrete types described below in the `messaging.knative.dev`
+and `eventing.knative.dev` API groups, Knative Eventing supports referencing
+objects in other API groups as destinations for event delivery. This is done by
+defining partial schemas which the other resources must support. The following
+interface contracts define resource fragments and partial schemas (required fields
+on an arbitrary API object) which form a basis for Knative Eventing.
+
+### Addressable
+
+**Addressable** resources expose a resource address (HTTP URL) in their `status`
```

Comment:
/address/`address`/  to make it clear that's the name of the property. And we should show an example to make it even more clear.

Reply:
Maybe I should link to https://github.com/knative/specs/blob/99bb2e0cb0e33f36dfd3a4475ebf08668fbca2f3/specs/eventing/control-plane.md#addressable-resolution from #25?